src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -207,6 +208,12 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
             c.argument('password', options_list=['--password', '-p'], help="The password to login to the custom registry. This can be plain text or a key vault secret URI.")
             c.argument('use_identity', help="The task managed identity used for the credential. Use '[system]' to refer to the system-assigned identity or a client id to refer to a user-assigned identity.")
 
+    with self.argument_context('acr task timer') as c:
+        # Timer trigger parameters
+        c.argument('timer_name', help="The name of the timer trigger.", required=True)
```

Comment:
maybe `--name` would be nicer instead of `acr task timer --timer-name`

Reply:
is --name used for task name?

Reply:
Yes, name is used for task name here already.